apps/mobile/src/app/wallet/create-new-wallet.tsx

```diff
@@ -10,27 +10,43 @@ import { Button } from '@/components/button';
 import { MnemonicWordBox } from '@/components/create-new-wallet/mnemonic-word-box';
 import { TransText } from '@/components/trans-text';
 import { APP_ROUTES } from '@/constants';
+import { tempMnemonicStore } from '@/state/storage-persistors';
 import { useTheme } from '@shopify/restyle';
 import { BlurView } from 'expo-blur';
 import * as Clipboard from 'expo-clipboard';
 import { useRouter } from 'expo-router';
 
+import { generateMnemonic } from '@leather.io/crypto';
 import { Box, Text, Theme, TouchableOpacity } from '@leather.io/ui/native';
 
-function getTestMnemonic() {
-  if (process.env.EXPO_PUBLIC_SECRET_KEY) {
-    return process.env.EXPO_PUBLIC_SECRET_KEY;
+function MnemonicDisplay({ mnemonic }: { mnemonic: string | null }) {
+  if (!mnemonic) {
+    return null;
   }
-  throw new Error('No mnemonic is provided in EXPO_PUBLIC_SECRET_KEY');
+  const mnemonicArr = mnemonic.split(' ');
```

Comment:
Wondering abt using `Arr` in naming, maybe something more descriptive would be better? We know it is an array from the method. `mnemoicWords` or something? Can ignore comment.

Reply:
+1 to favouring plural or something like MnemonicWordList to avoid naming as its js type